x/derivatives/keeper/lpt.go

```diff
@@ -83,6 +83,15 @@ func (k Keeper) DetermineMintingLPTokenAmount(ctx sdk.Context, amount sdk.Coin)
 	return sdk.NewCoin(types.LiquidityProviderTokenDenom, mintAmount.TruncateInt()), sdk.NewCoin(types.LiquidityProviderTokenDenom, mintAmount.Mul(mintFeeRate).TruncateInt()), nil
 }
 
+func (k Keeper) LdpDenomRate(ctx sdk.Context, redeemDenom string) (sdk.Dec, error) {
```

Comment:
What id Ldp?

DLP is the denom of LiquidityProviderToken.

How abou LiquidityProviderTokenDenomrate

Reply:
fixed